packages/app/ember-backstop/backstop.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-env browser, node */
```

Comment:
Is this file used?

Reply:
As discussed offline, we don't reference it but builds fail without it